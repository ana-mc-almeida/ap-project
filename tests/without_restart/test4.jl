
include("./common.jl")

handling(() -> print(reciprocal(0)),
    DivisionByZero => (c) -> println("I saw a division by zero"))