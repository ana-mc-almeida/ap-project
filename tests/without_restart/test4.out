I saw a division by zero
ERROR: DivisionByZero()